Samples/PackageWithExternalLocation/cppwinrt/PackageWithExternalLocationCppAppSetup/main.cpp

```diff
@@ -0,0 +1,85 @@
+﻿#include "pch.h"  
+#include <iostream>  
+#include <string>
+#include <winrt/Windows.Management.Deployment.h>  
+#include <winrt/Windows.ApplicationModel.Activation.h>  
+#include <shellapi.h>
+
+using namespace winrt;  
+using namespace winrt::Windows::Management::Deployment;  
+
+void InstallPackage(const std::wstring& packagePath, const std::wstring& externalLocation)
+{
+    try 
+    {
+        if (externalLocation.empty() || packagePath.empty())
+        {
+            MessageBox(nullptr, L"Failed to register app with identity. Please provide the correct absolute path to your package and external location", L"", MB_ICONINFORMATION);
+            return;
+        }
+        
+        winrt::Windows::Management::Deployment::PackageManager packageManager;
+        winrt::Windows::Management::Deployment::AddPackageOptions addOptions;
+        addOptions.ExternalLocationUri(winrt::Windows::Foundation::Uri(winrt::hstring(externalLocation)));
+
+        auto deploymentResult = packageManager.AddPackageByUriAsync(winrt::Windows::Foundation::Uri(winrt::hstring(packagePath)), addOptions).get();
+        auto extendedErrorCode = deploymentResult.ExtendedErrorCode();
+
+        if (extendedErrorCode != HRESULT_FROM_WIN32(ERROR_PACKAGES_IN_USE) && FAILED(extendedErrorCode))
+        {
+            wchar_t buff[10];
+            _itow_s(extendedErrorCode, buff, 16);
+            if (extendedErrorCode == HRESULT_FROM_WIN32(CERT_E_UNTRUSTEDROOT))
+            {
+                MessageBox(nullptr, L"Please install your certificate to Trusted Root Certification Authorities for Local Machine. Running without identity", L"buff", MB_ICONINFORMATION);
```

Comment:
```suggestion
                MessageBox(nullptr, L"Please install your certificate to Trusted Root Certification Authorities for Local Machine. Running without identity", buff, MB_ICONINFORMATION);
```

You only need `L""` around literal strings. Since `buff` is an array you just pass it as-is.